winpr/include/winpr/custom-crypto.h

```diff
@@ -265,9 +265,15 @@ extern "C"
 	WINPR_API void winpr_Cipher_Free(WINPR_CIPHER_CTX* ctx);
 
 	WINPR_ATTR_MALLOC(winpr_Cipher_Free, 1)
-	WINPR_API WINPR_CIPHER_CTX* winpr_Cipher_New(WINPR_CIPHER_TYPE cipher,
-	                                             WINPR_CRYPTO_OPERATION op, const void* key,
-	                                             const void* iv);
+	WINPR_API WINPR_DEPRECATED_VAR("[since 3.10.0] use winpr_Cipher_NewEx",
+	                               WINPR_CIPHER_CTX* winpr_Cipher_New(WINPR_CIPHER_TYPE cipher,
+	                                                                  WINPR_CRYPTO_OPERATION op,
+	                                                                  const void* key,
+	                                                                  const void* iv));
+	WINPR_ATTR_MALLOC(winpr_Cipher_Free, 1)
```

Comment:
Perhaps you should add the documentation so that we know that it's available since 3.10..

Reply:
it was not available before too (well, only if you used old OpenSSL)